src/main.js

```diff
@@ -37,10 +46,22 @@ const load = loader({
     setup: ({docs}) => docs.write({docsDir: process.env['DOCS_OUTPUT_DIR']}),
   },
 
-  server: {
-    requires: ['cfg', 'docs'],
-    setup: ({cfg, docs}) => {
-      console.log('Hello, world.');
+  succeedTaskQueue:{
+    requires: ['queue', 'cfg'],
+    setup: ({cfg, queue}) => new taskqueue.TaskQueue(cfg, queue, 'succeed'),
```

Comment:
I wonder if we should hardcode this like that. It's a nit, however, and doesn't block